api/applications/applicationRouter.js

```diff
@@ -353,27 +361,51 @@ router.post(
  *        $ref: '#/components/responses/UnauthorizedError'
  */
 
-// registers a user with okta and approves their application ticket
-
-router.put(
-  '/approve/:id',
-  authRequired,
-  adminRequired,
-  checkApplicationExists,
-  findProfile,
-  registerOktaUser,
-  (req, res, next) => {
-    const application_id = req.application.application_id;
+// Author: Christwide Oscar
+// Finding applicant and differentiating between mentors and mentees. Setting their application status to 'approved' from pending
+router.post('/approve/:profile_id', (req) => {
+  const { profile_id, low_income } = req.body;
+  if (typeof low_income !== undefined) {
```

Comment:
I like this solution--it's improved from the initial loom video. Even though it was working before, thanks for refactoring it to clean it up.